pages/three.js

```diff
@@ -30,9 +30,8 @@ var ThreePreview = function() {
 
         scene = new THREE.Scene();
 
-        defaultCamera = new THREE.PerspectiveCamera(45, container.offsetWidth / container.offsetHeight, 1, 20000);
+        defaultCamera = new THREE.PerspectiveCamera(45, container.offsetWidth / container.offsetHeight, 1e-5, 1e10);
```

Comment:
Those numbers seem quite magical.  Any chance you want to either give name them as variables for self-documentation, or add a comment for explicit documentation?